eZ/Publish/Core/REST/Server/Output/ValueObjectVisitor/ContentFieldValidationException.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/**
+ * File containing the ContentFieldValidationException ValueObjectVisitor class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
```

Comment:
To be skipped, same as [here](https://github.com/ezsystems/ezpublish-kernel/pull/1734/files#r73865889)